### ISSUE TITLE
File in Obsidian app showing hashed content (I think)

### ISSUE THREAD

**Issue by heywoodlh** [vrtmrz/filesystem-livesync#4]
So I think this is the final issue I need to work past and then I'll be good to use filesystem-livesync.

I create a new file named `tmp.md` with the following content: 

```
This is a pretty cool program
```

I see the watcher detects the change and syncs the file:

```
4/25/2022, 5:29:42 PM->tmp.md -> maybe new
4/25/2022, 5:29:42 PM->save ok:id:h:78376400 rev:1-e027f754356ef46c01ec4ac3b48fc614
4/25/2022, 5:29:42 PM->note content saven, pieces:1 new:1, skip:0, cache:0
4/25/2022, 5:29:42 PM->note saved:tmp.md:1-15b2a28b141d3a1ab413e19768bdef77
```

But here's how it looks in the Obsidian app: 

![tmp](https://user-images.githubusercontent.com/18178614/165143430-1bf1e0a9-03eb-4bd9-80c7-e20fc09997eb.png)

Here is my config: 

```
{
    "study": {
        "server": {
            "uri": "http://obsidian.kube/obsidian-study",
            "auth": {
                "username": "obsidian",
                "password": "<omitted>",
		"passphrase": ""
            },
            "initialScan": true 
        },
        "local": {
            "path": "/data/Study",
            "initialScan": true 
        },
        "auto_reconnect": true,
        "sync_on_connect": true 
    }
}

```

**Comment by vrtmrz** [comment]
Really thank you for testing!

It was encrypted empty string instead of skipping encryption.
Fixed at 24b3b7d8b13b596633a46a49b2628e86a7988784

**Comment by heywoodlh** [comment]
Fixed for me! Thanks for the quick change. Just an FYI, I had to completely remove the `server.auth.passphrase` field -- if you have it set to `""` like I did it still tried to encrypt the contents.
